histogramvec/histogramvec_test.go

```diff
@@ -329,3 +330,26 @@ func Test_Ensure(t *testing.T) {
 		})
 	}
 }
+
+// Test_Concurrency tests that concurrent access to a HistogramVec is safe.
+func Test_Concurrency(t *testing.T) {
```

Comment:
I assume there was just a case in which the code paniced because of concurrent map access?

Reply:
Yeah, with the fixes this no longer panics